frontend/src/views/website/website/create/index.vue

```diff
@@ -166,6 +166,7 @@
                                         <el-option label="Java" value="java"></el-option>
                                         <el-option label="Go" value="go"></el-option>
                                         <el-option label="Python" value="python"></el-option>
+                                        <el-option label=".NET" value="dotnet"></el-option>
                                     </el-select>
                                 </el-form-item>
                             </el-col>
```

Comment:
There doesn't appear to be any major differences between the two codes provided so far. However, it wouldn't hurt to double-check some specific details like formatting errors, unnecessary code sections, and inconsistencies with the data you're working on if those were there at all. Also, ensure that your variable names and other significant areas (like dates) align correctly throughout the document; it's important not to mix them up!